strawberryfields/apps/graph/resize.py

```diff
@@ -308,6 +308,18 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
+    .. code-block::
```

Comment:
Any specific reason why the recommended usage in sphinx is without the directive? Unless it may cause problems, it's probably best to leave `.. code-block::` for now

Reply:
I'm not sure it really matters too much, but for consistency, I would prefer if no `.. code-block::` was used for `>>>`, since the remaining SF documentation is written that way.

It is easy enough to add `.. doc-test::` where it is needed in a future PR (and indent)

Reply:
Thanks! Done!